src/test/java/hw_jdi/JdiTest.java

```diff
@@ -32,18 +35,7 @@ public void afterSuite() {
 
     @Test(dataProvider = "jdiSubmitMetalsAndColorsFormDataProvider")
     public void jdiSubmitMetalsAndColorsForm(MetalsAndColorsData metalsAndColorsData) {
-        JdiSite.open();
-        JdiSite.indexPage.login(User.ROMAN);
-        JdiSite.headerMenu.select(HeaderMenuData.METALS_AND_COLORS);
-
-        String[] summary = metalsAndColorsData.getSummary();
-        JdiSite.metalsAndColorsPage.selectSummary(summary[0], summary[1]);
-        JdiSite.metalsAndColorsPage.selectElements(metalsAndColorsData.getElements());
-        JdiSite.metalsAndColorsPage.selectColors(metalsAndColorsData.getColor());
-        JdiSite.metalsAndColorsPage.selectMetals(metalsAndColorsData.getMetals());
-        JdiSite.metalsAndColorsPage.selectVegetables(metalsAndColorsData.getVegetables());
-        JdiSite.metalsAndColorsPage.clickSubmitButton();
-
+        JdiSite.metalsAndColorsPage.submitJdiMetalsAndColorsForm(metalsAndColorsData);
         assertTrue(metalsAndColorsData.getNonDefaultResultStrings().containsAll(JdiSite.metalsAndColorsPage.getResult()));
```

Comment:
Basically, this is not the best idea to use assertTrue in this case. What are you gonna get as a error message in case of falure ?